report_common_nrq/report/common_template.xml

```diff
@@ -354,6 +354,23 @@
             </tr>
         </table>
     </template>
+    <template id="price_unit_display">
+        <t t-set="price_unit" t-value="l.price_unit" />
```

Comment:
We should probably leave this line outside of the template to leave the flexibility on what should be the value of `price_unit` (pay attention to the lines I highlighted in my last review).

Reply:
Thank you. I see.
I added comment to define t-set attr.